### ISSUE TITLE
Zoomable

### ISSUE THREAD

**Issue by keith6014** [nikolaydubina/treemap#30]
Are there any plans to make this zoomable?

**Comment by nikolaydubina** [comment]
yes, this is common major ask for this tool. I will see if I have time to add it. but feel free to add suggestions/PRs on how to do it as well meantime
